numpy/core/_add_newdocs.py

```diff
@@ -5599,13 +5680,31 @@
 
     These are roughly ordered from least-to-most precision.
 
+    Examples
+    --------
+
+    >>> dt = np.dtype(str)
+    >>> dt.num
+    19
+
+    >>> dt = np.dtype(float)
+    >>> dt.num
+    12
+
     """))
 
 add_newdoc('numpy.core.multiarray', 'dtype', ('shape',
     """
     Shape tuple of the sub-array if this data type describes a sub-array,
     and ``()`` otherwise.
 
+    Examples
+    --------
+
+    >>> dt = np.dtype('4i4')
+    >>> dt.shape
+    (4,)
```

Comment:
Can you also add `np.dtype(('i4', (2, 3)))` here, showing what more than 1d looks like?